documentation/feature/grenades.md

```diff
@@ -19,8 +19,12 @@ Adds stun grenade. This will also affect AI.
 
 ## Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings, 
-instructions. May not apply to all modules.
+### Switching between  throw modes
```

Comment:
These are features of ace_grenades and ace_weaponselect and won't work if those modules aren't present.